client/layouts/auth.vue

```diff
@@ -0,0 +1,57 @@
+<script lang="ts" setup>
+const color = useColorMode()
+const { t } = useI18n()
+
+const child = ref(null)
+const cardTransition = ref(false)
+
+// Since we cannot access transition hooks here, we have to listen to DOM mutations
+// to determine when the card is transitioning. When the card is doing so, a child
+// is added, and when the transition is over, the child is removed.
+// This should work as long as we don't mess with the DOM in other ways.
+useMutationObserver(child, (mutations) => {
```

Comment:
Since error and warning messages have been tested and nothing seemed to glitch nor animate strangely, I assume this solution is working and indeed not observing the DOM element deeply.